### PR TITLE
create vmOutput if error on system vm

### PR DESCRIPTION
**Pull request by sasurobert**
create vmOutput if error on system vm

### CODE REVIEW

File: vm/process/systemVM.go [hunk 1]
```diff
@@ -118,7 +119,18 @@ func (s *systemVM) RunSmartContractCall(input *vmcommon.ContractCallInput) (*vmc
...
 	returnCode := contract.Execute(input)
-	vmOutput := s.systemEI.CreateVMOutput()
+	vmOutput := &vmcommon.VMOutput{}
+	if returnCode == vmcommon.Ok {
+		vmOutput = s.systemEI.CreateVMOutput()
+	} else {
+		vmOutput = &vmcommon.VMOutput{
+			GasRemaining:  0,
+			GasRefund:     big.NewInt(0),
+			ReturnCode:    returnCode,
```
Comment (miiu96) [inline vm/process/systemVM.go:129]: this line can be removed because `ReturnCode` will be set at line `134`

**Review by miiu96 — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Review by miiu96 — APPROVED** [review]

File: vm/process/systemVM.go [hunk 2]
```diff
@@ -118,7 +119,17 @@ func (s *systemVM) RunSmartContractCall(input *vmcommon.ContractCallInput) (*vmc
 	}
 
 	returnCode := contract.Execute(input)
-	vmOutput := s.systemEI.CreateVMOutput()
+	var vmOutput *vmcommon.VMOutput
+	if returnCode == vmcommon.Ok {
```
Comment (iulianpascalau) [inline vm/process/systemVM.go:123]: we need an import-db process for this refactor

**Review by iulianpascalau — APPROVED** [review]

**Review by valentin-lup — APPROVED** [review]
Passed staggered import-db on May 13th.

**Review by gabi-vuls — APPROVED** [review]
System test passed.

**Pull request merged**